src/app.js

```diff
@@ -1,21 +1,46 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const fs = require('fs');
+const path = require('path');
+
+const [source, destination] = process.argv.slice(2);
+
+const moveFile = (src, dest) => {
+  if (!src || !dest) {
+    console.error('Source and destination must be specified');
+
+    return;
+  }
+
+  if (!fs.existsSync(src)) {
+    console.error(`Source file '${src}' does not exist`);
+
+    return;
+  }
+
+  let destPath = dest;
+
+  try {
+    const destStat = fs.fstatSync(fs.openSync(dest, 'r'));
+
+    if (destStat.isDirectory()) {
+      destPath = path.join(dest, path.basename(src));
+    }
+  } catch (err) {
+    if (err.code !== 'ENOENT') {
+      console.error(`Error checking destination file: ${err.message}`);
+
+      return;
+    }
+  }
+
+  try {
+    fs.renameSync(src, destPath);
+    console.log(`Moved '${src}' to '${destPath}'`);
```

Comment:
It would be nice to change the message `moved` or `renamed` to represent real action.